### PR TITLE
Disclaimer for experimental builds

### PR DESCRIPTION
**Pull request by nexusmrsep**
```SUMMARY: Interface "Disclaimer for experimental builds in MOTD"```

#### Purpose of change
Due to common misunderstanding of purpose and risks of using experimental builds, followed by lot of salty comments on forums, I deducted that there is a place for a disclaimer that will answer some popular questions and perhaps be a way to pass that knowledge in-game.

It is placed in MOTD section. This section is scrollable, so it can fit there fine.

#### Describe the solution
As above. The text of the disclaimer is available in the change itself. It's written in Q&A style for added readability. 

#### Describe alternatives you've considered
Answering the recurring questions again and again and again in forums.
Placing this in 'Help' section, but it's for game help not generalized disclaimers.

#### Additional context
N/A

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
Add some <color_x> tags for Q lines?

**Comment by Lorith** [comment]
Unfortunately I don't think this will help much, but it is certainly a good idea.  Perhaps also add a note that one should back up any saves they want to keep just in case?

**Comment by nexusmrsep** [comment]
@ZhilkinSerg I'll try to add colors tommorow.

@Lorith I'm well aware of that, but that's better than not having one. I've also thought of making an overkill version where all the experimentals would require you to physically accept the disclaimer popup once per version aka each time you update. But as I said that might be an overkill. And also can be done later if the need emerges. As for now, it will be a place to send someone who adamantly promotes an opinion of taking for granted that experimentals are being handed out nice and polished and guaranteed bugless. Whish is not the case as we know it.
I'll add the note you mentioned.

**Comment by davidpwbrown** [comment]
> @ZhilkinSerg I'll try to add colors tommorow.
> 
> @Lorith I'm well aware of that, but that's better than not having one. I've also thought of making an overkill version where all the experimentals would require you to physically accept the disclaimer popup once per version aka each time you update. But as I said that might be an overkill. And also can be done later if the need emerges. As for now, it will be a place to send someone who adamantly promotes an opinion of taking for granted that experimentals are being handed out nice and polished and guaranteed bugless. Whish is not the case as we know it.
> I'll add the note you mentioned.

Anyone who plays the experimental version has to sign a legal TOS with a phylactery of their blood sent to us, and if they complain about experimental being buggy / broken, we then have the legal right to perform a blood magic ceremony to make them smell bad forever and for hairs to sprout out of their noses in perpetuity.

File: data/motd/en.motd [hunk 1]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+
+******************************************************************************
+    Disclaimer for the experimental builds:
+******************************************************************************
+Q: What is an experimental build?
+A: The goal of the experimental builds is to be a second tier of quality
+control. It's the frontline where new content is released to be tested and
+balanced after initial quality control. Be aware that by choosing to play
+the latest experimental builds you are participating in the development
+process by literaly playtesting the new content, and you are prone to bugs
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
process by literally playtesting the new content, and it is prone to bugs
```

File: data/motd/en.motd [hunk 2]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+and underdeveloped features.
+
+Q: Why are experimental builds even a thing?
+A: Consider it like playing an alpha/beta version - bugs and underdeveloped
+features are part of it. Due to continuous integration rigors experimental
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
features are part of it. Due to continuous integration rigors, experimental
```

File: data/motd/en.motd [hunk 3]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+builds are always in a state of constant work in progress. Experimental is
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
builds are always in a state of a constant work in progress. Experimental is
```

File: data/motd/en.motd [hunk 4]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+a sandbox where the bug detecting and feature polishing takes place. Small
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
a sandbox where the bug detecting and feature polishing takes place. Big
```

File: data/motd/en.motd [hunk 5]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+and big features are constantly realeased to those builds to be checked
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
and small features are constantly released to those builds to be checked
```

File: data/motd/en.motd [hunk 6]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+for overall stability and balance. Because of that experimentals are not
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
for overall stability and balance. Because of that, experimentals are not
```

File: data/motd/en.motd [hunk 7]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+under the rigor of guaranteeing balance and stability. In fact, nothing
+is guaranteed in the experimental builds.
+
+Q: Are experimental builds playable then?
+A: Yes. But while experimental builds are being overall stable, it is also
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
A: Yes. But while experimental builds overall are stable, it is also 
```

File: data/motd/en.motd [hunk 8]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+builds are always in a state of constant work in progress. Experimental is
+a sandbox where the bug detecting and feature polishing takes place. Small
+and big features are constantly realeased to those builds to be checked
+for overall stability and balance. Because of that experimentals are not
+under the rigor of guaranteeing balance and stability. In fact, nothing
+is guaranteed in the experimental builds.
+
+Q: Are experimental builds playable then?
+A: Yes. But while experimental builds are being overall stable, it is also 
+not guaranteed, as features may be introduced in underdeveloped
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
not guaranteed as features may be introduced in an underdeveloped
```

File: data/motd/en.motd [hunk 9]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+state, in parts, and may introduce bugs, including severe ones that may
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
state, in parts; and may introduce bugs, including severe ones that may
```

File: data/motd/en.motd [hunk 10]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+crash the game or corrupt savegames. It's rare, but due to the complexity
+of the game, the initial quality control before merging with experimental
+build may be not be adequate, and there is always room for human error.
+This is why experimental builds exist in the first place.
+
+Q: Should I play experimental builds then, or stick to stable build?
+A: The reward is instant access to the newest content, the risk is losing
+your game progress if a nasty bug emerges. It is always your choice to play
+experimental builds, with stable build avalaible, so play at your own
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
experimental builds, with a stable build avalaible, so play at your own
```

File: data/motd/en.motd [hunk 11]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+discretion. You take all the burden if something goes wrong, so be warned.
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
discretion. You take all of the burden if something goes wrong, so be warned.
```

File: data/motd/en.motd [hunk 12]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+Take precausions, make backups. Sometimes it's even best to postpone
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
Take precautions, make backups. Sometimes it's even best to postpone
```

File: data/motd/en.motd [hunk 13]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+updating for few days until a big feature had been stabilized or
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
updating for few days until a big feature has been stabilized or
```

File: data/motd/en.motd [hunk 14]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+a significant bug had been removed.
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
a significant bug has been removed.
```

File: data/motd/en.motd [hunk 15]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+
+Q: Should I be aware of anything else?
+A: Yes. Remember that everything you see and are about to see is potentially
+subject to change. All features, including newly implemented, are a work in
+progress and may change in a course of the game development.
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
progress and may change in the course of game development.
```

File: data/motd/en.motd [hunk 16]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+And, as said previously, nothing is guaranteed, including balance and
+stability. 
+
+Q: What if I find the risk of playing experimental build unacceptable?
+A: If the risk is unacceptable - play the stable version. If you are willing
+to take the risk - enjoy the newly added content of the experimental builds,
+and parttake in development - report bugs, submit suggestions, or even
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
and partake in development - report bugs, submit suggestions, or even
```

File: data/motd/en.motd [hunk 17]
```diff
@@ -18,3 +18,55 @@ Join the discussion on forums and chats:
...
+Q: Should I be aware of anything else?
+A: Yes. Remember that everything you see and are about to see is potentially
+subject to change. All features, including newly implemented, are a work in
+progress and may change in a course of the game development.
+And, as said previously, nothing is guaranteed, including balance and
+stability. 
+
+Q: What if I find the risk of playing experimental build unacceptable?
+A: If the risk is unacceptable - play the stable version. If you are willing
+to take the risk - enjoy the newly added content of the experimental builds,
```
Comment (KorGgenT) [inline data/motd/en.motd]:
```suggestion
to take the risk - enjoy the newly added content of the experimental builds
```

**Pull request merged**